content/en/agent/basic_agent_usage/_index.md

```diff
@@ -256,13 +256,13 @@ Enabling JMX Checks forces the Agent to use more memory depending on the number
 {{< tabs >}}
 {{% tab "Agent v6 & v7" %}}
 
-* Agent Test version: 6.7.0
-* CPU: ~ 0.12% of the CPU used on average
-* Memory: ~ 60MB of RAM used (RSS memory)
-* Network bandwidth: ~ 86 B/s ▼ | 260 B/s ▲
+* Agent Test version: 7.34.0
+* CPU: ~ 0.08% of the CPU used on average
+* Memory: ~ 130MB of RAM used (RSS memory)
+* Network bandwidth: ~ 140 B/s ▼ | 800 B/s ▲
 * Disk:
-  * Linux 350MB to 400MB depending on the distribution
-  * Windows: 260MB
+  * Linux 850MB to 1.2GB depending on the distribution
+  * Windows: 870MB
 
 {{% /tab %}}
 {{% tab "Agent v5" %}}
```

Comment:
Could you remove this `Agent v5` tab entirely? So that there's no tab at all, only the Agent v7 numbers you worked on updating.

Reply:
Done in [d540b0c](https://github.com/DataDog/documentation/pull/13311/commits/d540b0ccc3eaed7a086abd3f32279363ca8b0f04)

Reply:
thanks!